src/pages/transaction/send.vue

```diff
@@ -422,6 +422,18 @@ export default {
       type: String,
       required: false,
     },
+    txid: {
```

Comment:
Ignore: This was restored on the https://github.com/paytaca/paytaca-app/commit/3d42291a3c4f492b5efca9a3e7ca88954f51766f